src/main/java/lotto/model/LottosBundleGenerator.java

```diff
@@ -0,0 +1,22 @@
+package lotto.model;
+
+import java.util.List;
+
+public class LottosBundleGenerator implements LottosGenerator {
+    private final Money money;
+    private final List<String> manualLottoText;
+
+    public LottosBundleGenerator(Money money, List<String> manualLottoText) {
+        this.money = money;
+        this.manualLottoText = manualLottoText;
+    }
+
+    @Override
+    public Lottos generate() {
+        Count manualCounts = new Count(manualLottoText.size());
+        Count autoCounts = money.countBuyableLottoTickets().subtract(manualCounts);
```

Comment:
```suggestion
        Count autoCounts = money.countBuyableLottoTickets().subtract(manualLottoText.size());
```
subtract에 Count를 인자로 받는 메서드도 제공하지만, int를 인자로 받는 메서드를 오버로딩한다면 위와 같이도 구현 가능함.
생성자만 오버로딩하지 말고 메서드에 대한 오버로딩도 추가해 보는 습관을 들이면 좋겠음.